compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -218,6 +218,9 @@ pub fn check_tied_features(
     sess: &Session,
     features: &FxHashMap<&str, bool>,
 ) -> Option<&'static [&'static str]> {
+    if features.len() == 0 {
```

Comment:
Nit: this is probably better written as something like:

```
if !features.is_empty() {
    for tied in ... {
        ...
    }
}
return None;
```

The code isn't particularly deeply nested here and avoids having to specify `None` as a return twice.